src/neon/how-tos/advance-analysis/undistort.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Undistorting Video and Gaze Data
+description: "Undistorting Neon's scene camera video and gaze data, using the intrinsic and extrinsic camera parameters."
+permalink: /neon/how-tos/advance-analysis/undistort/
+---
+# Correcting for the fish-eye lens distortion on Neon's scene camera
+One particular challenge that often arises in eye tracking setups is the presence of [fish-eye lens distortions](https://en.wikipedia.org/wiki/Fisheye_lens), also known as barrel distortions. Fish-eye lenses, with their wide-angle characteristics, allow for a broad field of view, enabling the capture of more information within a single frame. However, this advantage comes at the cost of introducing non-linear distortions. 
+
+While we account for them in Pupil Cloud, and we even give you the possibility to download the undistorted video using the [Gaze Overlay enrichment](/enrichments/gaze-overlay), this is not the case when using the [Realtime API](/neon/real-time-api) or raw data, where you may want to correct it by yourself.
+
+## Why is it important to correct for fisheye lens distortions?
```

Comment:
Showing a side-by-side of distorted vs undistorted somewhere like in the analogue how-to for undistorting PI data would be great!